### PR TITLE
Graduate multiple sizes huge pages to GA

### PR DESCRIPTION
**Pull request by bart0sh**
This PR is a part of PR series to graduate support for multiple sizes hugepages to GA.
 
Support for multiple sizes huge pages was implemented in 1.18 and graduated to beta in 1.19.
Community feedback was positive since then, so it's time to graduate it to GA.

k8s PR:  kubernetes/kubernetes#99144
KEP PR: kubernetes/enhancements#2534

### CODE REVIEW

**Comment by tengqm** [comment]
I believe this PR should go to the dev-1.21 branch rather than master.

**Comment by bart0sh** [comment]
@tengqm thank you! rebased.

**Comment by tengqm** [comment]
/lgtm
Looks to me there is only one page where the feature gate is mentioned.

**Comment by irvifa** [comment]
/hold
Until kubernetes/kubernetes#99144 is merged

**Comment by reylejano** [comment]
Please see comment https://github.com/kubernetes/kubernetes/pull/99144#issuecomment-795659896
>kubernetes/enhancements#1539 is currently not being tracked by the release team since it was not opted in before the enhancements freeze. Also with the code freeze deadline being yesterday, for this to be considered into the release, you'll need to request an exception. If you have any questions, please reach out on slack #sig-release. Thanks!

The related kubernetes/enhancements#2534 is not tracked for 1.21
https://github.com/kubernetes/enhancements/pull/2534#issuecomment-795661347

**Comment by reylejano** [comment]
Closing the PR because the related kubernetes/enhancements#2534 is not tracked for 1.21
Please see comment kubernetes/kubernetes#99144 (comment)

> kubernetes/enhancements#1539 is currently not being tracked by the release team since it was not opted in before the enhancements freeze. Also with the code freeze deadline being yesterday, for this to be considered into the release, you'll need to request an exception. If you have any questions, please reach out on slack #sig-release. Thanks!

See kubernetes/enhancements#2534 (comment)

**Comment by bart0sh** [comment]
@reylejano this PR is targeting 1.22. Please, reopen. I'll update all related PRs.

**Comment by reylejano** [comment]
/hold until PR targets the `dev-1.22` branch and 
Enhancements kubernetes/enhancements#2534 is tracked for 1.22
k8s PR kubernetes/kubernetes#99144 has merged
/assign @PI-Victor

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (ehashman) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:128]: This second line should be added to the graduated/deprecated table below.

**Review by ehashman — CHANGES_REQUESTED** [review]
Needs rebase.

**Comment by sftim** [comment]
@bart0sh how's this PR going? Does this feature need documentation changes for v1.22?

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (ehashman) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:128]: This has been fixed in 372d34dc9467c9448a8e6a08a92e6b94260eeac5

**Comment by ehashman** [comment]
PR still needs docs, waiting on final approvals for feature PR to merge.

@bart0sh can you please rebase?

**Comment by bart0sh** [comment]
@reylejano 
> /hold until the following:
> 
> * PR targets the `dev-1.22` branch

done

> * Enhancements [Graduate multiple sizes huge pages to GA enhancements#2534](https://github.com/kubernetes/enhancements/pull/2534) is tracked for 1.22

done

> * k/k PR [promote huge page storage medium size to GA kubernetes#99144](https://github.com/kubernetes/kubernetes/pull/99144) has merged

done 

Please unhold review and merge, thanks.

**Comment by bart0sh** [comment]
@sftim @ehashman sorry for the delay, I was on vacation.

@sftim Yes, these changed need to be merged into 1.22. Please review & merge.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (bart0sh) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:130]: I've added all 3 lines as it's done for HugePages feature.

Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:130]: These lines are for a GA'ed feature, so they are supposed to be MOVED to the second table, not just COPIED.

Comment (bart0sh) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: @tengqm Thank you for pointing out to it. Done.

**Comment by tengqm** [comment]
/lgtm
Leaving the approval and unhold to release team.

**Review by ehashman — COMMENTED** [review]
/lgtm
for SIG Node

**Comment by PI-Victor** [comment]
thank you all for checking!

/hold cancel
/approve

**Pull request merged**
